grassroots-backend/src/auth/Auth.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from "@nestjs/testing";
```

Comment:
This (and the test below) are just autogenerated. Should I leave them in place, or remove them unless we actually have some behavior we want to unit test?